modules/utilityFunction.js

```diff
@@ -0,0 +1,127 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export function loadBookslist(bookInstanceFromBookStore) {
+  function loopingBook(book) {
+    return `<div id="${book.id}" class="book-card">
+                  <p class="book-details">
+                      "${book.title}" 
+                      by 
+                      ${book.author}
+                  </p>
+                  <button class="remove-button" data-book="${book.id}">Remove</button>
+              </div>`;
+  }
+
+  const bookContainer = document.querySelector('.books-container');
+  bookContainer.innerHTML = bookInstanceFromBookStore.store.map((book) => loopingBook(book)).join('');
+
+  // remove button
+
+  const removeButtons = document.querySelectorAll('.remove-button');
+  removeButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      bookInstanceFromBookStore.remove(id);
+      loadBookslist(bookInstanceFromBookStore);
+      localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+    });
+  });
+}
+
+export function loadContacts(contactInstanceFromContact) {
+  function loopingContacts(contact) {
+    return `<div class="contact-card">
```

Comment:
- [ ] The keyword `function` is depreciated in the new javascript ES6 syntax and is replaced by the arrow functions. So I would suggest you to use  `arrow functions` instead of function keyword as well. 

Example snippet:

```js

export const loadContacts = (contactInstanceFromContact) => {
...
}

```